components/Form/index.jsx

```diff
@@ -29,48 +60,97 @@ function Form({ onSubmit, expense = [], isEditMode, setToast }) {
       <StyledTitle>{isEditMode ? "Edit Expense" : "Add Expense"}</StyledTitle>
       <form onSubmit={onSubmit}>
         <StyledMargin>
+          <StyledLabel htmlFor="name">Expense Title</StyledLabel>
           <StyledInput
+            $isValid={isWrong.name ? false : true}
```

Comment:
Here we can pass `isWrong.name` directly to `$isValid` as the value should be either `true` or `false`

```suggestion
            $isValid={isWrong.name}
```